src/spasht/ui/template/show_definitions.html.twig

```diff
@@ -35,16 +32,13 @@
         <td></td>
         <td id="definitionsRow_provider{{ loop.index0 }}">{{ result.provider }}</td>
         <td>
-          <li>
-            <a href="#" onclick="showDetails({{ loop.index0 }})">Show Details</a>
-          </li>
+            | <a href="#">Show Details</a>

```

Comment:
Instead of using pipe (`|`), can this be done with CSS? This look odd IMO

Reply:
Will change soon